data_relay.py

```diff
@@ -41,7 +41,6 @@
         ('B', 'gpsStatus'),
         ('B', 'batteryLevel'),
         ('B', 'waypointCount'),
-	('B', 'airspeed'),
```

Comment:
I forgot whether it worked or didn't work when I added the one byte of padding. I'll verify before merging.